Source/Data/SpectrumAnalysis/KTSeqLine.cc

```diff
@@ -0,0 +1,273 @@
+/*
+ * KTScoredSpectrum.cc
+ *
+ *  Created on: Mar 10, 2016
+ *      Author: Christine
+ */
+
+#include <KTSeqLine.hh>
+
+#include "KTLogger.hh"
+
+#include <iostream>
+#include <cmath>
+
+
+namespace Katydid
+{
+	KTLOGGER(seqlog, "KTSeqLine");
+
+
+
+    LineRef::LineRef(const double& InitialSlope):
+        fStartTimeInRunC(0.0),
+        fStartTimeInAcq(0.0),
+        fEndTimeInRunC(0.0),
+        fEndTimeInAcq(0.0),
+        fStartFrequency(0.0),
+        fEndFrequency(0.0),
+        fInitialSlope(InitialSlope),
+        fSlope(0.0),
+        fNPoints(0),
+        fComponent(0),
+        fAmplitudeSum(0.0),
+        fAcquisitionID(0),
+        fProcTrackMinPoints(0),
+        fProcTrackAssError(0.),
+        fIntercept(0.),
+        fSlopeSigma(0.),
+        fInterceptSigma(0.),
+        fStartFrequencySigma(0.),
+        fEndFrequencySigma(0.)
+        {}
+
+    LineRef::~LineRef()
+    {}
+
+    void LineRef::InsertPoint(const Point& Point)
+    {
+        KTDEBUG(seqlog, "Insert point");
+        fTrimmingLimits.push_back(Point.fThreshold); //new_trimming_limits);
+        fAmplitudeList.push_back(Point.fAmplitude);
+
+        LinePoint newPoint(Point.fBinInSlice, Point.fPointFreq, Point.fTimeInAcq, Point.fTimeInRunC, Point.fAmplitude, Point.fThreshold, Point.fAcquisitionID, Point.fComponent);
+        fLinePoints.push_back(newPoint);
+
+        this->UpdateLineParameters();
+        this->CalculateSlope();
+    }
+
+    void LineRef::FinishTrack()
```

Comment:
Even if this isn't used, please fix the indentation down in the second half of the function.